sklearn/metrics/cluster/unsupervised.py

```diff
@@ -102,7 +107,7 @@ def silhouette_score(X, labels, metric='euclidean', sample_size=None,
 
 
 def silhouette_samples(X, labels, metric='euclidean', **kwds):
-    """Compute the Silhouette Coefficient for each sample.
+    r"""Compute the Silhouette Coefficient for each sample.
```

Comment:
I thought it was a rule of pep8 but in fact it's a rule of the pep257: if you display some latex equation you've to add an r before the docstring.
I'll remove that.